monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage.js

```diff
@@ -474,7 +474,7 @@ class RunMonkeyPageComponent extends AuthComponent {
           {
             this.state.isErrorWhileCollectingAwsMachines ?
               <div style={{'marginTop': '1em'}}>
-                <p class="alert alert-warning">
+                <p class="alert alert-danger">
                   <i className="glyphicon glyphicon-warning-sign" style={{'marginRight': '5px'}}/>
                   Error while collecting AWS machine data. Error message: <code>{this.state.awsMachineCollectionErrorMsg}</code><br/>
                   Are you sure you've set the correct role on your Island AWS machine?<br/>
```

Comment:
This comment is about "Not sure what this is? ..."
I agree it should be above the component regardless (like what you did)
But why does it also appear on the error?

Reply:
It's two different boxes, with different texts - just a little similar :) 
See line 240 for the info box. 
This, unlike 240, is a part of the error text which aims to help the user get to the documentation quickly.